### DISCUSSION TITLE
Background color contrast issue

### DISCUSSION THREAD

**Discussion by davulrich** [ohmyzsh/ohmyzsh#11200] [Help]
Hi,

I would like to ask you for help. I use iTerm2 and ohmyzsh with powerpack10k and I face the issue that the background colors have slightly different contrast at the sides.

![image](https://user-images.githubusercontent.com/46351885/192839549-49d7af05-5229-4bae-9e33-1be00c433de4.png)

I spent 2 hours googling and playing with settings with no success.

I checked couple of Github issues as well:
- https://github.com/ohmyzsh/ohmyzsh/issues/4246
- https://github.com/ohmyzsh/ohmyzsh/issues/4352

But with the same result.  Interesting is that in VS code terminal it looks ok:
![image](https://user-images.githubusercontent.com/46351885/192842684-b28e6ec5-decb-4d16-b6c7-551ef068aa74.png)

Did anybody face this issue? 

Thanks in advance for you help.

OS: MacOS 12.5.1
iTerm2: 3.4.16

**Comment by mcornella** [comment]
Check whether you have different colors for white in foreground and background. This is a powerlevel10k issue though, if that doesn't fix it you should look in https://github.com/romkatv/powerlevel10k.
